### PR TITLE
Copy ome.api.* documentation to omero.api.* (3)

### PR DESCRIPTION
**Pull request by sbesson**
See https://trello.com/c/KLCaZX4X/19-copy-ome-api-documentation-to-omero-api

Follow-up of #4547 and https://github.com/openmicroscopy/openmicroscopy/pull/4569, this PR copies more of the ome.api.* Javadoc documentation into the slice classes.

To review this PR, either build the slice documentation via release-slice2html or use the published CI documentation or OMERO.docs* artifacts. Check the links are all working and the method parameters match.

### CODE REVIEW

**Comment by jburel** [comment]
Usage of  ``""`` e.g. ``All "draft" shares`` might generate problems with ice3.6 cf. https://github.com/openmicroscopy/openmicroscopy/pull/4452#issuecomment-209314946

File: components/blitz/resources/omero/api/IRenderingSettings.ice [hunk 1]
```diff
@@ -20,30 +20,341 @@ module omero {
...
+                 * @param pTo The pixels set to copy the settings to.
+                 * @return See above.
+                 **/
                 idempotent bool sanityCheckPixels(omero::model::Pixels pFrom, omero::model::Pixels pTo) throws ServerError;
+
+                /**
+                 * Returns the default rendering settings for a given pixels
+                 * for the current user.
+                 *
+                 * @param pixelsId The Id of the <code>Pixels</code>
```
Comment (ximenesuk) [inline components/blitz/resources/omero/api/IRenderingSettings.ice]: This file makes fairly liberal use of code blocks to highlight class names, etc.

**Comment by ximenesuk** [comment]
Looks reasonable. It's noticeable that the ISession and IShare docs make very little use of params and return values but that's all for another day! 

The use of code blocks in IRenderingSessings could be reviewed here or pushed till later.

**Comment by sbesson** [comment]
I'll try and fix the `code` blocks  in `IRenderingSettings` to match the proposal I made in https://github.com/openmicroscopy/openmicroscopy/pull/4569#issuecomment-208989924.

File: components/blitz/resources/omero/api/IRenderingSettings.ice [hunk 2]
```diff
@@ -154,207 +153,206 @@ module omero {
...
-		 * @throws ValidationException if an illegal <code>type</code>
+                 * @throws ValidationException if an illegal <code>type</code>
                  *         is used.
-		 */
+                 */
                 omero::sys::LongList resetDefaultsByOwnerInSet(string type, omero::sys::LongList nodeIds) throws ServerError;
 
-		/**
-		 * Resets a the channel windows for one or many containers
+		        /**
```
Comment (ximenesuk) [inline components/blitz/resources/omero/api/IRenderingSettings.ice]: This and a few other opening comment lines look indented too far.

Comment (sbesson) [inline components/blitz/resources/omero/api/IRenderingSettings.ice]: Good catch - see ecc4c1d79b3b396dd0b0705022c213ec7b632ad8

**Comment by ximenesuk** [comment]
Looks good to merge.

**Pull request merged**